docs/sources/configure-server/anonymous-usage-statistics-reporting.md

```diff
@@ -0,0 +1,60 @@
+---
+
+description: Learn about Grafana Pyroscope anonymous usage statistics reporting
+menuTitle: Anonymous usage statistics reporting
+title: About Grafana Pyroscope anonymous usage statistics reporting
+weight: 30
+---
+
+# About Grafana Pyroscope anonymous usage statistics reporting
+
+Grafana Pyroscope includes a system that optionally and anonymously reports non-sensitive, non-personally identifiable information about the running Pyroscope cluster to a remote statistics server.
```

Comment:
Similar like in the admonition, maybe we also include the "By default" in the first sentence.